data-prepper-core/src/main/java/org/opensearch/dataprepper/pipeline/Pipeline.java

```diff
@@ -177,12 +172,10 @@ public void shutdown() {
      * 4. Stopping the ProcessWorkers if they are unable to exit gracefully
      * 5. Shutting down processors and sinks
      * 6. Stopping the sink ExecutorService
-     *
-     * @param processorTimeout the maximum time to wait after initiating shutdown to forcefully shutdown process worker
      */
-    public void shutdown(int processorTimeout) {
-        LOG.info("Pipeline [{}] - Received shutdown signal with timeout {}, will initiate the shutdown process",
-                name, processorTimeout);
+    public void shutdown() {
+        LOG.info("Pipeline [{}] - Received shutdown signal with processor shutdown timeout {}, will initiate the shutdown process",
+                name, processorShutdownTimeoutInMillis);
```

Comment:
I think even this would be a good place to have a `Duration`. We could give a nicer string here (e.g. 2m30s).